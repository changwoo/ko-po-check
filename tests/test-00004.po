# test 3
#
msgid ""
msgstr ""
"Project-Id-Version: test 3\n"
"POT-Creation-Date: 2002-07-14 21:52+0900\n"
"PO-Revision-Date: 2002-07-14 23:42+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=EUC-KR\n"
"Content-Transfer-Encoding: 8bit\n"

#: test.c:1
msgid "test_name"
msgstr "%s°¡ ¾øÀ¾´Ï´Ù\""
