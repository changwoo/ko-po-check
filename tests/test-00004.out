test-00004.po:18: copyright notice는 번역하면 안 됩니다
test-00004.po:23: copyright notice는 번역하면 안 됩니다
test-00004.po:28: copyright notice는 번역하면 안 됩니다
